README.md

```diff
@@ -66,14 +66,21 @@ export KUBE_TEST_REPO_LIST=$(pwd)/repo_list
 
 Once those are set, you could run all the `[SIG-Windows]` tests with:
 
-```
-./e2e.test --provider=skeleton --ginkgo.noColor --ginkgo.focus="\[sig-windows\]" --node-os-distro="windows"
+```bash
+./e2e.test \
+    --provider=skeleton
```

Comment:
missing continuation `\`

Reply:
I think it was a one-liner in case folks are coming from windows and in powershell prompt but could make note to use wsl?

Reply:
Right, I will change it back to one-line